src/main/resources/config/liquibase/changelog/20231120095701_changelog.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="andreas (generated)" id="1700470627638-3">
+        <createTable tableName="math_exercise_details">
+            <column name="example_solution" type="VARCHAR(255)"/>
+            <column name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="math_exercise_detailsPK"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="andreas (generated)" id="1700470627638-4">
```

Comment:
Not sure why you split this off into a separate changeset, as it could also just be a single one, but if you keep it separate the same applies here
```suggestion
    <changeSet author="andreas" id="20231120095701-2">
```

Reply:
Auto-generated by liquibase, as it truly syncs w/ the application model.

Reply:
Auto generated does not necessarily mean good. The author is not really what we usually use within Artemis and the id is not up to our guidelines. 
You can of course use tools to autogenerate this, but please also do a manual check of it afterwards.